docs/workflows/verify-images.rst

```diff
@@ -0,0 +1,262 @@
+.. _verify-images:
+
+Configure client signature verification policy
+==============================================
+
+In order to verify container images consumed from Pulp Container Registry the following files need
+to be configured on the client side:
```

Comment:
This introduction sounds like a list is expected. Can you maybe make a bullet list for those files and their description?